tests/VolleyManagement.UnitTests/Services/GameReportService/GetPivotStandingsTests.cs

```diff
@@ -409,5 +469,45 @@ public void GetPivotStandings_PlannedGamesWithoutResults_StandingsContainPlanned
         {
             AssertTournamentStandingsAreEqual(expected, actual, message, new PivotStandingsComparer());
         }
+
+        private void AssertPivotStandingsAreEqualByPoints(
+           TournamentStandings<PivotStandingsDto> expected,
+           TournamentStandings<PivotStandingsDto> actual,
+           string message)
+        {
+            TeamStandingsDtoComparer comparer = new TeamStandingsDtoComparer();
+            comparer.WithPointsComparer();
+            AssertTournamentStandingsAreEqual(expected, actual, message, new PivotStandingsComparer(comparer));
+        }
+
+        private void AssertPivotStandingsAreEqualBySet(
+           TournamentStandings<PivotStandingsDto> expected,
+           TournamentStandings<PivotStandingsDto> actual,
+           string message)
+        {
+            TeamStandingsDtoComparer comparer = new TeamStandingsDtoComparer();
+            comparer.WithSetRatioComparer();
+            AssertTournamentStandingsAreEqual(expected, actual, message, new PivotStandingsComparer(comparer));
+        }
+
+        private void AssertPivotStandingsAreEqualByBalls(
+          TournamentStandings<PivotStandingsDto> expected,
+          TournamentStandings<PivotStandingsDto> actual,
+          string message)
+        {
+            TeamStandingsDtoComparer comparer = new TeamStandingsDtoComparer();
```

Comment:
Why do you do it on this instance and pass it in PivotStandings comparer? Can you configure PivotStandingsComparer?